### ISSUE TITLE
Spring rest docs with Streaming Responses

### ISSUE THREAD

**Issue by cristianprofile** [spring-projects/spring-restdocs#669]
To **test infinite streams** (for example, "text/event-stream" or "application/stream+json") using **webtestclient**, I need to exit the chained API (by using returnResult). Spring documentation recommends using the StepVerifier from the reactor-test module to do that, as the following example shows:

```
val eventFlux = result.getResponseBody()
StepVerifier.create(eventFlux) .expectNext(person) .expectNextCount(4)
.consumeNextWith { p -> ... } .thenCancel()
.verify()
```
Is there any feature in the roadmap to be able to create Spring rest docs documentation using this approach?

**Comment by wilkinsona** [comment]
Duplicates #219. @cristianprofile please feel free to comment on that issue with any requirements you have, what you're particularly interested in, etc.

**Comment by izeye** [comment]
This issue seems to be meant to be closed as a duplicate of #219 based on the Andy's comment.

**Comment by wilkinsona** [comment]
Thanks, @izeye.
